MdePkg/Library/VsIntrinsicLib/IA32/llmul.asm

```diff
@@ -0,0 +1,98 @@
+;***
+;llmul.asm - long multiply routine
+;
+;       Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
No `All rights reserved.` any longer, can you check for the other files?

Reply:
Removed all - thanks